lib/beaker/dsl/install_utils.rb

```diff
@@ -708,8 +708,12 @@ def install_puppet(opts = {})
             end
           end
 
-          # Certain install paths may not create the config dirs/files needed
-          on host, "mkdir -p #{host['puppetpath']}" unless host[:type] =~ /aio/
+          puppetpaths = host['puppetpath'].split(';')
```

Comment:
This setting is not defined in AIO, and will try to split on nil. Also this is a windows specific separator character, but this method is called in other contexts.